website/blog/2023-12-08-silver.md

```diff
@@ -0,0 +1,585 @@
+
+## Use presentation compiler for synthetic decorations
+
+Up until this release Metals would use the semanticdb information for displaying
+synthetic decorations such as inferred types or implicit conversions. This was
+not ideal as it meant that the information was not always up to date and could
+be inconsistent with the current state of code.
+
+![synthetics](https://i.imgur.com/td11vRH.gif)
+
+To fix that [jkciesluk](https://github.com/jkciesluk) switched to using the
+presentation compiler for this information. This means that each time code
+changes the decorations will be refreshed. This also added a number of new
+decorations that were not previously available for Scala 3 and it will be on par
+with the Scala 2 ones.
+
+## Base Metals view on indexing information
+
+Metals tree view enables users to navigate their workspace in a more structured
+way by directly showing the build modules and their packages, classes, objects
+etc. It would also allow users to navigate their dependency jars. Previously,
+this was based on special Scala 2 signature information, which means it wouldn't
+work in Scala 3.
+
+From this release we changed it to use the same information that is used for
+indexing in case of jars and semanticdb for local workspace files. This means
+that Scala 3 is fully supported and this has been the last big feature missing
+for full parity between Scala 2 and 3.
+
+![tree-view](https://i.imgur.com/Vun4kJh.gif)
+
+All the existing features should now work for Scala 3 as well. This includes the
+reveal command which allows to quickly reveal the current file in the tree view
+including inside jars.
+
+![tree-view-reveal](https://i.imgur.com/xW0W3I3.gif) Additionally, we improved
+icons used by the tree view. They will now use the default VS Code icons for the
+most part, which should make them more consistent with the rest of the editor.
+
+## Go to definition for scaladoc
+
+When writing scaladoc for your methods or classes you can use links, which are
+of a form `[[name.name.ClassName]]`. Thanks to
+[kasiaMarek](https://github.com/kasiaMarek) Metals will now allow you to
+navigate to the target of the link via normal go to definition.
+
+![scaladoc-def](https://i.imgur.com/TYNMCEm.gif)
+
+To learn more about linking in scaladoc check out the
+[Scala documentation](https://docs.scala-lang.org/scala3/guides/scaladoc/linking.html)
+
+## Custom project root setting
+
+Metals currently will try to find the proper build tool root for your project,
+which is useful especially if you open a parent directory that doesn't contain a
+build tool configuration. However, this can be problematic if you have a less
+traditional setup and you need to be able to control the root directory. For
+example if you have an sbt source dependency and you still want to have that
+dependency open in the same window.
+
+To help with that we added a new setting `metals.customProjectRoot` that allows
```

Comment:
Not breaking, but maybe mention this down below for plugin authors as well to draw extra attention that they should add it, since I haven't added it yet to nvim-metals even 😬

Reply:
Done